src/Bridge/Symfony/Bundle/Resources/config/metadata.xml

```diff
@@ -74,6 +89,27 @@
             <argument type="service" id="api_platform.metadata.property.metadata_factory.validator.inner" />
         </service>
 
+        <service id="api_platform.metadata.property.metadata_factory.cached" class="ApiPlatform\Core\Metadata\Property\Factory\CachedPropertyMetadataFactory" decorates="api_platform.metadata.property.metadata_factory" decoration-priority="-1" public="false">
```

Comment:
The speed can be largely improved (after the first request) by adding the `ApcuAdapter` in prod mode.

But maybe can we wait for symfony/symfony#17868.